test/e2e/crd_e2e_test.go

```diff
@@ -261,8 +261,9 @@ var _ = Describe("CRD Versions", func() {
 		})).Should(Succeed())
 		ctx.Ctx().Logf("updated subscription to point to alpha channel")
 
+		checker := subscriptionStateAtLatestChecker()
 		subscriptionAtLatestWithDifferentInstallPlan := func(v *operatorsv1alpha1.Subscription) bool {
-			return subscriptionStateAtLatestChecker(v) && v.Status.InstallPlanRef != nil && v.Status.InstallPlanRef.Name != fetchedInstallPlan.Name
+			return checker(v) && v.Status.InstallPlanRef != nil && v.Status.InstallPlanRef.Name != fetchedInstallPlan.Name
```

Comment:
`checker` seems redundant? What's this change for?

Reply:
I changed the signature - it's got one more level of nesting to the closure now.

Reply:
So `checker(v)` is `subscriptionStateAtLatestChecker()(v)`